### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by oliiiiiiiiiiiii**
Reached Me -> Reach Me

love -> loves

### CODE REVIEW

**Comment by MarzaElise** [comment]
reach urmum

**Comment by MarzaElise** [comment]
btw add [hutch bot](https://dsc.gg/hutch-bot) to your server

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Hello There!
 
-My name is TheGenocides. You can call me Geno or Luke. I'm a 14 years old male python developer that love making projects. I'm also a twitter enthusiast and a gamer addicted! 
+My name is TheGenocides. You can call me Geno or Luke. I'm a 14 years old male python developer that loves making projects. I'm also a twitter enthusiast and a gamer addicted!
```
Comment (MarzaElise) [inline README.md:3]: Add a comma after `I'm a 14 years old`. Also, the sentence `gamer addicted!` kind of doesnt make sense to me

**Comment by Genoinno** [comment]
> reach urmum

Not if i reach yours first 😆

**Comment by MarzaElise** [comment]
why delet my comment

**Comment by MarzaElise** [comment]
hutch bot very kool, if u disagree, ur racist and homophobic

**Comment by Genoinno** [comment]
> why delet my comment

Because its dumb, i like your mom better then your bot. Advertise your mom next time 😃

**Comment by Genoinno** [comment]
> hutch bot very kool, if u disagree, ur racist and homophobic

Disagree because both of you are nob

**Comment by oliiiiiiiiiiiii** [comment]
How aam i a nub ?
@TheGenocides 
:suseyes:

**Comment by Genoinno** [comment]
> How aam i a nub ?
> 
> @TheGenocides 
> 
> :suseyes:

You're not

**Comment by UnrealFar** [comment]
e

**Pull request merged**